src/parameter_server.cpp

```diff
@@ -139,6 +160,35 @@ int main(int argc, char** argv) {
     << rank << " rank"
     << std::endl;
 
+  std::string ps_ip_string = FLAGS_ps_ip;
+  std::vector<std::string> ps_ips;
+  std::string tmp;
+  std::stringstream ss(ps_ip_string);
+  while (ss) {
+    if (!getline(ss, tmp, ','))
+      break;
+    ps_ips.push_back(tmp);
+  }
+
+  std::string ps_port_string = FLAGS_ps_port;
```

Comment:
We should refactor this into a separate function

Reply:
Fixed